### PR TITLE
Add integration tests

### PR DESCRIPTION
**Pull request by adejanovski**
Ready for review and merge.

### CODE REVIEW

File: cstar/command.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def _parse(command_name, filename, definition):
                 arguments.append(Argument(**json.loads(value)))
             else:
                 warn("Ignoring unknown property %s while parsing %s" % (name, filename))
-
+
```
Comment (rzvoncek) [inline cstar/command.py:83]: Just a blank space, we could clean up.

Comment (rzvoncek) [reply cstar/command.py:83]: There's actually a few of these across the PR.

File: tests/docker-compose.yml [hunk 2]
```diff
@@ -1,85 +1,132 @@
-version: '3.8'
+version: '3.2'
```
Comment (rzvoncek) [inline tests/docker-compose.yml:1]: Is there any specific reason for degrade?

**Review by rzvoncek — APPROVED** [review]
One question though: is it still possible to run the ITs locally? Looks like it is, just wanna be sure.

File: tests/docker-compose.yml (hunk 2 above)
Comment (adejanovski) [reply tests/docker-compose.yml:1]: yes, CircleCI's docker-compose version wouldn't accept higher than 3.2.

**Comment by adejanovski** [comment]
> One question though: is it still possible to run the ITs locally? Looks like it is, just wanna be sure.

Sure thing, just cd into `tests` and run `make all`.

File: cstar/command.py (hunk 1 above)
Comment (adejanovski) [reply cstar/command.py:83]: ok, will fix this.

File: tests/Makefile [hunk 3]
```diff
@@ -1,24 +1,40 @@
 
 all: setup test teardown
-	
+
 
 test:
-	docker exec -t tests_cstar_1 cstar run --command='nodetool status' --seed-host cassandra-0 --ssh-password=root
+	docker exec -t tests_cstar_1 bash -c "cd /usr/src/app/cstar/ && behave tests/features -D SEED_HOST=cassandra-10 --no-capture --no-capture-stderr --format=plain --junit --junit-directory=/usr/src/app/cstar/tests/results"
```
Comment (arodrime) [inline tests/Makefile:6]: @adejanovski that's the part I was missing/not knowing how to plug! 

Thanks for taking over on this @adejanovski (LGTM)

**Review by arodrime — APPROVED** [review]

**Pull request merged**